app/models/services/service_instance.rb

```diff
@@ -134,7 +138,7 @@ def credentials_with_serialization
     alias_method_chain :credentials, 'serialization'
 
     def in_suspended_org?
-      space.in_suspended_org?
+      space && space.in_suspended_org?
```

Comment:
@jenspinney @deniseyu  

I'm not sure of all of the places where this is called, but I'm a bit worried that this will allow users to do things with the service instance when the org is suspended that they shouldn't be able to do.

E.g. Org One binds some email sending service to Space A and shares it with Space B in Org Two. If Org One is spamming people with their email sending service and is suspended will this mean that Org Two can keep on making changes to the service instance and using it since `space` will be nil and always return `false`?

Reply:
Thanks for the feedback, @tcdowney!

This method appears to only be called in [service_instance_access.rb](https://github.com/cloudfoundry-incubator/cloud_controller_ng_sapi/blob/service-instance-sharing/app/access/service_instance_access.rb).

As a general rule, if space is `nil`, the user may only do read-ish things (nothing involving modification, deletion, etc.) on the service instance because they don't have access to the originating space of the service instance. The places where we check whether the org is suspended are modification-type access checks, so we'll fail those anyway because space is nil. In our version of service_instance_access, you'll see that wherever this method is called, we've made changes to also explicitly check if the space is available. 

If we reversed this and returned true for `service_instance.in_suspended_org?` when space is nil, it seems counterintuitive that it would be asserting that the org of the service instance is suspended even though we know nothing about the originating org. I would imagine if `service_instance.in_suspended_org?` returned true, I could return some error message like, "The org you're in has been suspended", which is not true in this case.

Thanks!
Jen and @deniseyu